File: WebApp/WebApp/Models/SoldTicket.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Web;

namespace WebApp.Models
{
    public class SoldTicket
    {
        [Key]
        public Guid Id { set; get; }
        [Required]
        public string Type { set; get; } //"vremenska", "godisnja".... name iz product typea
        [Required]
        public string UserId { get; set; }
        public ApplicationUser User { get; set; }
        public DateTime Expires { get; set; }
        public int Usages { get; set; } = 0;
        public Double Price { get; set; }
        [Required]
        public DateTime DateOfPurchase { get; set; }
    }
}

File: angular-app/src/models/sold-ticket.ts
import { Guid } from "guid-typescript";
import { User } from './user';

export class SoldTicket {
    public id: Guid;
    public type: string;
    public userId: Guid;
    public user: User;
    public expires: Date;
    public usages: number;
    public price: number;
    public dateOfPurchase: Date;
}